app/helpers/application_helper.rb

```diff
@@ -22,4 +22,11 @@ def render_flash_message(flash_value)
     end
   end
 
+  def user_has_membership_application?
+    current_user.try(:membership_applications).try(:any?) && current_user.try(:membership_applications).try(:last).try(:persisted?)
```

Comment:
Add line above this:
```
return false if current_user.nil?
```
(that is, if you keep this method .... which I recommend against - see comments below)